src/Row/index.jsx

```diff
@@ -38,7 +69,26 @@ Row.propTypes =
      */
     gutters       : PropTypes.oneOf( [ 'none', 'S', 'M', 'L' ] ),
     /**
-     *  Row role
+     * Wrap content
+     */
+    hasWrap       : PropTypes.bool,
+    /**
+     *  onClick callback function:
+     *  ( e ) => { ... }
+     */
+    onClick       : PropTypes.func,
+    /**
+     *  onMouseOut callback function:
+     *  ( e ) => { ... }
+     */
+    onMouseOut    : PropTypes.func,
+    /**
+     *  onMouseOver callback function:
+     *  ( e ) => { ... }
+     */
+    onMouseOver   : PropTypes.func,
```

Comment:
Since we’re cleaning things up here I think we can remove any event props, I think they’re probably unnecessary